src/main/java/com/google/api/generator/gapic/composer/samplecode/ServiceClientSampleCodeComposer.java

```diff
@@ -761,6 +770,78 @@ public static String composeStreamCallableMethodHeaderSampleCode(
     return bodyStatements;
   }
 
+  private static List<Statement> composeStreamBidiSampleCodeBodyStatements(
+      Method method, VariableExpr clientVarExpr, AssignmentExpr requestAssignmentExpr) {
+    List<Expr> bodyExprs = new ArrayList<>();
+
+    // Create bidi stream variable expression and assign it with invoking client's bidi stream
+    // method.
+    // e.g. BidiStream<EchoRequest, EchoResponse> bidiStream = echoClient.chatCallable().call();
+    TypeNode bidiStreamType =
```

Comment:
Since this method is already named `bidi`, do the variable names still need `bidi` in them?

Reply:
I used `bidiStream` to match the original sample code syntax, but I am open to the variable name. A stream variable in server is `stream`, so I think it make sense use `stream` instead of `bidiStream`. Are you also preferring using `stream` instead?

Reply:
If it's for matching the variable name, then `bidiStream` LGTM.